src/Simulator/types.ts

```diff
@@ -0,0 +1,26 @@
+import { NetConnectionParams, IClientSimulatorImpl } from "../simularium";
+import { ISimulariumFile } from "../simularium/ISimulariumFile";
+
+export interface BaseSimulatorParams {
+    fileName: string;
+}
+
+export interface RemoteSimulatorParams extends BaseSimulatorParams {
+    netConnectionSettings?: NetConnectionParams;
+    requestJson?: boolean;
+    prefetchFrames?: boolean;
+}
+
+export interface ClientSimulatorParams extends BaseSimulatorParams {
```

Comment:
just curious, why do these all extend BaseSimulatorParams?  it feels like BaseSimulatorParams is never going to grow beyond fileName and ClientSimulatorParams doesn't even have a fileName.. 
Is BaseSimulatorParams used anywhere outside of this file?